src/map/utils/petutils.cpp

```diff
@@ -1096,8 +1096,10 @@ namespace petutils
 
         Pet_t* PPetData = g_PPetList.at(PetID);
 
-        if (PMaster->objtype == TYPE_PC)
+        if (PMaster->objtype == TYPE_PC){
```

Comment:
Its ok to leave open while you work on it and if you figure it out just updating your branch will automatic update the pull request. *Sometimes* we'll close out really old ones we think got abandoned.